python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -0,0 +1,96 @@
+"""
+Django settings for cac_tripplanner project.
```

Comment:
Yeah, I was just about to say the same thing. It'd be good to keep it in line with how we do it in other projects unless there's a good reason to change it.